console/src/assets/i18n/de.json

```diff
@@ -1692,7 +1692,7 @@
     "AZUREADTENANTTYPES": {
       "0": "Common",
       "1": "Organizations",
-      "2": "Customers"
+      "2": "Consumers"
```

Comment:
Why not Single-Tenant, Multi-Tenant and Personal Accounts?

Reply:
Because it is not necessary true. Its about Common is if you allow all accounts. Organizations is single or multi tenant and consumer are personal accounts